### DISCUSSION TITLE
Extract bundle translations with +intl-icu

### DISCUSSION THREAD

**Discussion by a-vallet** [symfony/symfony#48185] [Q&A] [answered]
Hello, 

I've a question about translation, we're using Loco provider to manager translations in our project but in Symfony 5.4 there are bugs with +intl-icu domain and others.

We've currently all our domains with +intl-icu expecting validators and security who came from bundle.
So when we pull from Loco if we put +intl-icu option we get all domains with the suffix and its creating security+intl-icu and validators+intl-icu not overwriting initial domain who still be used.

My question is, it is possible to extract initially those domains with +intl-icu to have all domains on the same format ?

Thanks,

**Comment by MatTheCat** [comment, marked as answer]
`+intl-icu` is just a suffix so it does not change the domain.

It does however imply another message format so you cannot just switch between the two. So if you want to use the ICU message format for every domain, you have to convert and commit those who are not.
